package.json

```diff
@@ -19,20 +19,20 @@
 	"devDependencies": {
 		"@commitlint/cli": "^17.0.3",
 		"@commitlint/config-conventional": "^17.0.3",
-		"@discordjs/collection": "^1.0.1",
+		"@discordjs/collection": "^1.1.0",
 		"@favware/cliff-jumper": "^1.8.7",
 		"@favware/npm-deprecate": "^1.0.5",
 		"@favware/rollup-type-bundler": "^1.0.11",
 		"@sapphire/framework": "^3.0.0",
 		"@types/lodash": "^4.14.184",
-		"@types/node": "^18.7.8",
+		"@types/node": "^18.7.11",
 		"@types/ws": "^8.5.3",
-		"@typescript-eslint/eslint-plugin": "^5.33.1",
-		"@typescript-eslint/parser": "^5.33.1",
+		"@typescript-eslint/eslint-plugin": "^5.34.0",
+		"@typescript-eslint/parser": "^5.34.0",
 		"@vitest/coverage-c8": "^0.22.1",
 		"cz-conventional-changelog": "^3.3.0",
-		"discord-api-types": "^0.33.5",
-		"discord.js": "^13.10.2",
+		"discord-api-types": "^0.37.4",
```

Comment:
dapi-types can be removed now as it's reexported from djs.

Reply:
Keeping it for editor autocompletes